### PR TITLE
Add SPM support

### PR DESCRIPTION
**Pull request by eddie-zhang**

### CODE REVIEW

**Comment by BetweenTwoBits** [comment]
@cwisecarver, any chance of getting this PR looked at soon? Our team plans to integrate soon, and we'd love to have SPM support from the start.

**Comment by dbaldwin-nbc** [comment]
@abelsonlive @cwisecarver Are there any plans to merge this PR or add SPM support in another release?

**Comment by abelsonlive** [comment]
@BetweenTwoBits @eddie-zhang  Could you provide a brief description to PR of how we can verify this change works and doesn't cause regressions?

**Comment by abelsonlive** [comment]
Thanks @eddie-zhang ! Could you point to a guide that outlines this?

>  The repo would need to have releases tagged with git tags in addition, in order to support SPM

**Comment by abelsonlive** [comment]
Thanks @eddie-zhang. The release is here: https://github.com/Parsely/AnalyticsSDK-iOS/releases/tag/0.2.1.  Let me know if that works for you!

**Comment by BetweenTwoBits** [comment]
@abelsonlive why close the PR rather than merge it? I think we'd all assume that you'd keep supporting people implementing via SPM

**Comment by abelsonlive** [comment]
@BetweenTwoBits  I added support via: https://github.com/Parsely/AnalyticsSDK-iOS/commit/c1a3bd02ff85baf8b79aff0893876eee0e594ae6

Here's the release: https://github.com/Parsely/AnalyticsSDK-iOS/releases/tag/0.2.1

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
```
[the remark this answers is not in the document]
Comment (cwisecarver) [reply Package.swift:1]: Sorry @eddie-zhang, we've got separate work on-going to add SPM support. I've reverted these commits for now. I'll @ you on the PR once we're ready to merge it.

**Pull request closed without merging**